lib/manageiq/appliance_console/cli.rb

```diff
@@ -175,10 +187,19 @@ def parse(args)
         opt :oidc_unconfig,        "Unconfigure Appliance OpenID-Connect Authentication",            :type => :boolean, :default => false
         opt :server,               "{start|stop|restart} actions on evmserverd Server",   :type => :string
         opt :openscap,             "Setup OpenScap", :type => :boolean, :default => false
-        opt :messaging_hostname, "Messaging Hostname", :type => :string
-        opt :messaging_username, "Messaging Username", :type => :string
-        opt :messaging_password, "Messaging Password", :type => :string
-        opt :messaging_port,     "Messaging Port",     :type => :integer
+        opt :message_server_config,       "Configure Appliance as a Kafka Message Server",               :type => :boolean, :default => false
+        opt :message_server_unconfig,     "Unconfigure Appliance as a Kafka Message Server",             :type => :boolean, :default => false
```

Comment:
We should probably have something like https://github.com/ManageIQ/manageiq-appliance_console/pull/151/commits/e4a22ff4116f0b53081a1bc22b6ef2203fa46e66#diff-53f36d8a1e1eb8aa2021a33997666d2713b68d425ab75a84c5e7899a1a7db895R204 that exits if the user provides config and unconfig for server or client

Reply:
Thank you @bdunne Good idea.

Address with commit https://github.com/ManageIQ/manageiq-appliance_console/pull/151/commits/ac6ad794dced6e27cab9cf1643b5c4787e1becdb